zigate/adminpanel/__init__.py

```diff
@@ -43,17 +44,43 @@ def get_url(routename, **kargs):
     @app.route('/', name='index')
     @bottle.view('index')
     def index():
-        from zigate import version
-        print(zigate_instance)
         connected = zigate_instance.connection and zigate_instance.connection.is_connected()
-        return {'port': zigate_instance._port or 'auto',
-                'libversion': version.__version__,
-                'version': zigate_instance.get_version_text(),
-                'connected': connected,
-                'devices': zigate_instance.devices,
```

Comment:
In my first experiments, I altered the `devices` and `groups` in the `index.tpl` template and noticed it actually modified the values of the `zigate_instance.devices` and `zigate_instance.groups`, breaking the zigate component and corrupting the `zigate.json` file. Hopefully, I had a not too old backup of this file.